library/general/test/linuxrc_test.rb

```diff
@@ -237,31 +237,41 @@ def load_install_inf(defaults_replacement = {})
         .with(path(".etc.install_inf"), nil)
     end
 
-    context "when vnc will be disabled" do
+    context "when vnc should be disabled" do
       it "updates install.inf" do
         expect(Yast::SCR).to receive(:Write)
-          .with(path(".etc.install_inf.VNC"), 0)
+          .with(path(".etc.install_inf.VNC"), 0).twice
```

Comment:
It took me a while to realize why it is written twice, it looked wrong at the first sight.

Please add a comment that we are testing two services ("vnc" and "VNC"), the same below.

Reply:
Okaj